src/client/public/runner.ts

```diff
@@ -171,7 +187,9 @@ interface InitializationParams {
 
     /** Creates a snippet iframe and returns it (still hidden). Returns true on success
      * (e.g., snippet indeed shown, in contrast with, say, the Trust dialog being shown, but not the snippet) */
-    function replaceSnippetIframe(html: string, officeJS: string, isTrustedSnippet: boolean): boolean {
+    function replaceSnippetIframe(params: { html: string, officeJS: string, isTrustedSnippet: boolean, isMaker: boolean }): boolean {
```

Comment:
FYI, this DOES have a way of being done in TypeScript in a *somewhat* prettier way

    function replaceSnippetIframe(
        {html, officeJS, isTurstedSnippet, isMaker}
        : {html: string, officeJS: string, isTrustedSnippet: boolean, isMaker: boolean }
    ): boolean {
         ...
    }

Reply:
is that really better? you have to type all the params twice..